content/articles/ms-ignite-2018.md

```diff
@@ -13,70 +13,42 @@ logo: /img/logo-sm.svg
 ![](http://appanacdn.blob.core.windows.net/cdn/icons/aic.png)
 
 ### Introduction
+Just three months ago [we announced that Microsoft joined the OpenCensus project](https://open.microsoft.com/2018/06/13/microsoft-joins-the-opencensus-project/), and last week at [Microsoft Ignite](https://www.microsoft.com/ignite) we were able to share a few more details about how OpenCensus will be leveraged and advanced.
 
-Just 3 month passed since the announcement that Microsoft is joining the
-OpenCensus project. This week at [Microsoft
-Ignite](https://www.microsoft.com/ignite) a Open Census were part of
-many conversation. It makes the goal of building a standardized platform
-for instrumenting metrics and distributed traces that work across
-various programming languages and technologies closer.
+### OpenCensus spotlight
 
-### Keynote
-
-As part of a keynote Microsoft announced support of Virtual Nodes by
-Azure Kubernetes Service. Virtual Nodes allow to scale your app faster,
-in a matter of minutes. What's the best way to demonstrate this if not
-the Application Insights [Live
+During one of Ignite's keynotes, Microsoft announced support for Azure Kubernetes Service (AKS) Virtual Nodes. Virtual Nodes allow applications to scale more quickly, in a matter of mere minutes. The demonstration included a highlight of Azure Monitor for Application's [Live
 Metrics Stream](https://docs.microsoft.com/azure/application-insights/app-insights-live-stream).
 
-![Microsoft Ignite 2018 keynote AKS Virtual Nodes with Application
-Insights and Open Census](ms-ignite-2018-images/ms-ignite-kyenote.gif).
+[![Microsoft Ignite 2018 keynote AKS Virtual Nodes with Application
+Insights and Open Census](ms-ignite-2018-images/ms-ignite-kyenote.gif)](https://mediastream.microsoft.com/events/2018/1809/Ignite/player/tracks/track2.html?start=17300)
 
-Link to the [video](https://mediastream.microsoft.com/events/2018/1809/Ignite/player/tracks/track2.html?start=17300).
+The keen eyed among you may have noticed that this demo is powered by [OpenCensus](https://github.com/Azure-Samples/virtual-node-autoscale)!
 
-Looking at demo application you may see that it is powered by [Open
-Census](https://github.com/Azure-Samples/virtual-node-autoscale).
+![OpenCensus in Ignite Keynote](ms-ignite-2018-images/keynote-demo-app-uses-open-census.png)
 
-![](ms-ignite-2018-images/keynote-demo-app-uses-open-census.png)
+As mentioned in the demo itself, OpenCensus's vendor neutral instrumentation provides users the freedom to either leverage the features and power built into Azure Monitor, or to easily provide their own tooling within Azure. This is the benefit of OpenCensus and its community oriented approach.
```

Comment:
OpenCensus wasn't mentioned in the demo. The openness of Azure was

Reply:
Fixed.